src/App.tsx

```diff
@@ -1,12 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { GoodList } from './components/GoodsList';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
+import { getAll, get5First, getRedGoods } from './api/goods';
+// // or
 // import * as goodsAPI from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const App: React.FC = () => {
+  const [selectedGoods, setSelectedGoods] = useState<Good[]>([]);
+
+  return (
+    <div className="App content is-large has-text-centered">
+      <h1 className="title is-2 mt-2 mb-0">Dynamic list of Goods</h1>
+      <GoodList goods={selectedGoods} />
+
+      <button
+        type="button"
+        className="button is-medium is-primary mx-2"
+        onClick={async () => setSelectedGoods(await getAll())}
```

Comment:
Will be better to store the result of functions in some variable and set set it to selected goods